src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -9,6 +10,27 @@ public class RegistrationServiceImpl implements RegistrationService {
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null) {
+            throw new RegistrationValidationException("User cannot be null");
+        }
+        if (user.getLogin().length() < 6) {
+            throw new RegistrationValidationException("Login must contain at least 6 characters");
```

Comment:
The check for login length does not account for null login. If `user.getLogin()` returns null, calling `.length()` will result in a NullPointerException. Consider adding a null check before this condition.